python/src/aiconfig/editor/client/src/LocalEditor.tsx

```diff
@@ -17,7 +17,11 @@ import { ufetch } from "ufetch";
 import { ROUTE_TABLE } from "./utils/api";
 import { streamingApiChain } from "./utils/oboeHelpers";
 
-export default function Editor() {
+type Props = {
+  readOnly?: boolean;
+};
+
+export default function Editor({ readOnly = false }: Props) {
```

Comment:
```suggestion

export default function Editor() {
```

LocalEditor shouldn't have `readOnly` as a prop since the component that is being packaged and reused is the `AIConfigEditor` component. LocalEditor will never be readonly